modules/nw-networkpolicy-about.adoc

```diff
@@ -120,6 +120,31 @@ spec:
       port: 443
 ----
 
+* Accept connections by using both namespace and Pod selectors:
++
+To match network traffic by combining namespace and Pod selectors, you can use a NetworkPolicy object similar to the following:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
```

Comment:
apiVersion is now ```networking.k8s.io/v1```. This also needs to be updated in all releases and current one.

Reply:
@anuragthehatter, do you mean all releases of OCP?

Reply:
@jboxman I noticed all earlier releases already have "networking.k8s.io/v1" so that seems good. Just want to make sure we don't have "extensions/v1beta1" somewhere